src/stores/PreferenceStore.ts

```diff
@@ -132,7 +132,7 @@ export class PreferenceStore {
     @observable beamColor: string;
     @observable beamType: BeamType;
     @observable beamWidth: number;
-    @observable regionContainer: RegionStore;
+    readonly regionContainer: RegionStore;
```

Comment:
this variable will be removed after server preference branch, can keep it unmodified to avoid conflict